### ISSUE TITLE
loss INF in training

### ISSUE THREAD

**Issue by danqu130** [MCG-NJU/CamLiFlow#2]
Thank you for opensource this wonderful work!
When I tried to train your model, I found that the loss appeared inf in the first things pretraining stage.
I use 4 3090 GPUs and the pytorch version is 1.9.1+cu111 with python3.8.
I haven't changed the data preprocessing and training code, have you ever had this problem before?

Here is the training log. 
[train.log](https://github.com/MCG-NJU/CamLiFlow/files/8598798/train.log)
![image](https://user-images.githubusercontent.com/15975091/166143659-5cb38158-522f-47a0-b76c-fe337b048742.png)

**Comment by afterthat97** [comment]
Looks like the loss explodes during the training. This sometimes happens and you can resume training from the last checkpoint that didn't explode (sorry for the inconvenience):

```
python train.py --config conf/train/pretrain.yaml --weights outputs/pretrain/ckpts/epoch-xxx.pt --resume
```

**Comment by danqu130** [comment]
Your answer has solved my problem, thank you!
